src/app/client/src/app/modules/shared/components/page-section/page-section.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="ui link popular-section" *ngIf="section.contents && section.contents.length > 0">
     <h4 class="ui header card-section-header sectionHeading">{{section.name}}
         <span class="ui red circular inline label">{{section.contents.length}}</span>
+        <button *ngIf="section.contents.length >= 10 && section.name != My Courses " type="button" (click)="navigateToViewAll(section)">more</button>
```

Comment:
they want view all for my course as well right

Reply:
No, they only want it for other courses which have the length of courses more than or equal to 10.